Escape App.Utils.renderer.default()

raffaelj opened the pull request:
Escape `<` and `>` in `App.Utils.renderer.default()` to prevent wrong opened/closed html elements inside raw field.

It seems, that other characters doesn't need to be escaped, because riot.js does that jpb internally.

It would escape wysiwyg contents, so I added a simple wysiwyg renderer, that truncates words.

aheinze commented:
This is the wrong approach, you should do the escaping in `App.Utils.renderer.wysiwyg`  
`App.Utils.renderer.default` should stay as it is

raffaelj commented:
The wysiwyg field didn't need escaping, but I left the renderer with the truncated word string with stripped tags.

I also reverted the change to the default renderer and added a text and a textarea renderer instead. Now they both are escaped.

Before escaping the text field:
![cp-renderer-specialchars-before](https://user-images.githubusercontent.com/13042193/77102870-d2b49c80-6a19-11ea-9155-e6f34169372d.png)

After escaping:
![cp-renderer-specialchars-after](https://user-images.githubusercontent.com/13042193/77102874-d3e5c980-6a19-11ea-8336-caa3b0836639.png)

aheinze commented:
Thanks Raffael!
